docs/cli.md

```diff
@@ -94,10 +98,79 @@ With `--all` flag, you create config file including all default setting of offic
       executable_methods:
         - method_name: "node_governance_alarm"`
 ```
-
 Also, You can set the home directory of VATZ (default "~/.vatz") with `--home` flag.
 And you can create the new config file with your disired name with `--output` flag
 
+## Start
+You can start VATZ when you finish set up config default.yaml or your own config yaml file.
```

Comment:
You can start VATZ after setting up the default.yaml or your own config YAML file.